src/olympia/reviewers/utils.py

```diff
@@ -1016,25 +991,65 @@ def reject_multiple_versions(self):
         latest_version = self.version
         self.version = None
         self.files = None
-        action_id = (amo.LOG.REJECT_CONTENT if self.content_review
-                     else amo.LOG.REJECT_VERSION)
-        timestamp = datetime.now()
-        log.info(
-            u'Making %s versions %s disabled' % (
-                self.addon,
-                u', '.join(str(v.pk) for v in self.data['versions'])))
+        now = datetime.now()
+        if self.data.get('delayed_rejection'):
+            pending_rejection_deadline = now + timedelta(
+                days=int(self.data['delayed_rejection_days'])
+            )
+        else:
+            pending_rejection_deadline = None
+        if pending_rejection_deadline:
+            action_id = (amo.LOG.REJECT_CONTENT_DELAYED if self.content_review
+                         else amo.LOG.REJECT_VERSION_DELAYED)
+            log.info(
+                'Marking %s versions %s for delayed rejection' % (
```

Comment:
I know it wasn't there before, but is there any value in adding an indicator of whether this is a content or version rejection to this log message?

Reply:
I don't think it matters much. We can find the info later if we want in `ActivityLog`, this logging is more useful to say "we've gone through the code that sets delayed rejection flag, if it's not there something is wrong".